### ISSUE TITLE
Improvement: Make the "Manage this page"-link optional

### ISSUE THREAD

**Issue by addvanced** [goksan/statusnook#15]
I don't want it to be easily accessible for anyone to go to the login page.
So, it would be nice if there was an option to remove the "Manage this page"-link on the status page.

**Comment by goksan** [comment]
Thanks for raising this.

How would users get to the login page when the link is optional? Directly via the URL?

**Comment by addvanced** [comment]
@goksan exactly. :) I think most people would know the URL for the admin panel after being logged in the first time.

**Comment by addvanced** [comment]
@goksan > well.. I managed to make a PR ( #16 ). Maybe you can use it - or maybe just for inspiration :)
